src/pages/home/report/ReportActionCompose/Suggestions.tsx

```diff
@@ -124,6 +124,11 @@ function Suggestions(
         suggestionEmojiRef.current?.setShouldBlockSuggestionCalc(shouldBlock);
         suggestionMentionRef.current?.setShouldBlockSuggestionCalc(shouldBlock);
     }, []);
+    const checkIfSuggestionVisible = useCallback((): boolean => {
```

Comment:
```suggestion
    const isSuggestionsMenuVisible = useCallback((): boolean => {
```

Reply:
As was discussed above, I changed it to `getIsSuggestionsMenuVisible`